server.js

```diff
@@ -52,6 +54,12 @@ if (config.IS_PRODUCTION) {
   app.enable('trust proxy')
 }
 
+// Ensure we have a useful transaction id
```

Comment:
This would be cleaner and more obvious if it was a transactionID middleware, e.g.
```
app.use(setTransactionId`

Reply:
Yes it totally would.

Reply:
and stuff like this be moved out too ;)

```
if (config.SENTRY.DSN) {
  Sentry.init({
    dsn: config.SENTRY.DSN,
    environment: config.SENTRY.ENVIRONMENT,
    release: config.SENTRY.RELEASE,
  })

  app.use(
    Sentry.Handlers.requestHandler({
      // Ensure we don't include `data` to avoid sending any PPI
      request: ['cookies', 'headers', 'method', 'query_string', 'url'],
      user: ['id', 'username', 'permissions'],
    })
  )
  app.use(sentryRequestId)
}
```

Reply:
and done